organization/_posts/2022/2022-03-24-generators.md

```diff
@@ -0,0 +1,30 @@
+Minutes of the HSF Generator WG Meeting No:18 Thursday 24 March 2022 https://indico.cern.ch/event/1141896/?note=192096
```

Comment:
Hi @efeyazgan 

You need to put a header at the top of the file to get it to be parsed correctly. See any of the other minutes files from recent meetings.

Something like:

```
---
title: "HSF Coordination Meeting #225, 17 March 2022"
layout: plain_toc
---
```